spec/services/create_fund_activity_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe CreateFundActivity do
+  let(:organisation) { create(:organisation) }
+
+  describe "#call" do
+    it "successfully creates a Fund Activity" do
+      result = described_class.new(organisation_id: organisation.id).call
+
+      expect(result.organisation).to eq(organisation)
```

Comment:
I think there is value in splitting these assertions out into their own isolated unit tests here. Doing so can help us document what each part is responsible for which in turn helps us know where to make future changes with confidence, without affecting other parts. Individual tests also allow us to wrap them in context blocks for unhappy paths. 

```
it "associates the new activity to the organisation passed in"
it "sets the form wizard to the first step"
it "sets the level as type 'fund'"
it "stores information about the organisation that funds this activity"
```